common/lib/xmodule/xmodule/tests/test_course_module.py

```diff
@@ -142,6 +143,16 @@ def test_may_certify(self):
         self.assertFalse(self.future_noshow_certs.may_certify())
 
 
+class CourseSummaryHasEnded(unittest.TestCase):
```

Comment:
CourseSummary has very few tests, and the ones that do exist are spread around (not in this file, though CourseSummary is defined in course_module.py). Adding this to existing tests didn't seem to make sense.